sgp4/export.py

```diff
@@ -0,0 +1,101 @@
+"""
+Export orbit data to a Two-Line-Element representation.
+"""
+from sgp4.io import compute_checksum
+
+from sgp4.model import Satrec
+
+
+def export_tle(satrec):
+    """
+     Generate the TLE for the data in the `Satrec` object.
+
+    Forms the two string lines of the TLE and returns them as a Tuple.
+
+    Parameters
+    ----------
+    satrec : Satrec
+        Object that holds the orbit information
+
+    Returns
+    -------
+    (line1, line2) : (str, str)
+    """
+
+    # Define constants
+    from math import pi
+    deg2rad = pi / 180.0  # 0.0174532925199433
+    xpdotp = 1440.0 / (2.0 * pi)  # 229.1831180523293
+
+    # --------------------- Start generating line 1 ---------------------
+
+    # Build the list by appending successive items
+    line_as_list = ["1 "]
+
```

Comment:
Go ahead right here and say:
```
append = line_as_list.append
```

Then all the subsequent calls can just say `append()` instead of repeating the name `line_as_list`. Oh, and, if you'd like, feel free to call it just `strings` or `pieces`. Readers typically will even with a simple single-word name recognize the maneuver and know what's going on. (And you can do the same down where you work on line 2.)

Reply:
I feel like I'm getting a one-to-one training from a Python expert, cheers for that. I know my way around Java, but I'm new to all things Pythonic. :)